src/js/shop/components/Checkout/Checkout.js

```diff
@@ -0,0 +1,259 @@
+import camelCase from "lodash.camelcase";
+import isObject from "lodash.isobject";
+import set from "lodash.set";
+import React, { useEffect } from "react";
+import PropTypes from "prop-types";
+import {
+    NavLink as RRNavLink,
+    Navigate,
+    Routes,
+    Route,
+    useNavigate,
+    useLocation,
+    useHref,
+} from "react-router-dom";
+import { FormattedMessage } from "react-intl";
+import classNames from "classnames";
+import { useImmerReducer } from "use-immer";
+
+import { SHOP_ROOT } from "../../../constants";
+import { Account, Address, Payment } from ".";
+import { EMPTY_ADDRESS } from "./constants";
+import { checkAddressFieldsComplete } from "./utils";
+
+const getActiveNavClassNames = ({ isActive, enabled = false }) =>
+    classNames("navigation__link", {
+        "navigation__link--active": isActive,
+        "navigation__link--enabled": enabled,
+    });
+
+const camelize = (obj) => {
```

Comment:
Is this for formatting API data? I thought it'd be easier to return formatted response from backend.

Reply:
temporary solution, I'll add a comment. At some point I want to add drf-camelcase renderer on the backend, but that will mean that _all_ the JS has to be updated from snake_case to camelCase which is quite a big impact.

Doing the camelization here allows it to not leak into all the nested components while using JS conventions (= avoiding snake_case in the child components) without having to micro-manage it.